From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 21 Aug 2019 14:08:52 -0400
Subject: types: Unarmor 76-character-long lines

When 76-char base64 lines were present, PGPy was failing to de-armor
them.

Closes: #276
Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
(cherry picked from commit 75beb08733e8c51a2fc4af5abd5eaa6f52d2b4af)
---
 pgpy/types.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pgpy/types.py b/pgpy/types.py
index 37add5e..2decb29 100644
--- a/pgpy/types.py
+++ b/pgpy/types.py
@@ -66,7 +66,7 @@ class Armorable(six.with_metaclass(abc.ABCMeta)):
                          (?P<headers>(^.+:\ .+(?:\r?\n))+)?(?:\r?\n)?
                          # capture all lines of the body, up to 76 characters long,
                          # including the newline, and the pad character(s)
-                         (?P<body>([A-Za-z0-9+/]{1,75}={,2}(?:\r?\n))+)
+                         (?P<body>([A-Za-z0-9+/]{1,76}={,2}(?:\r?\n))+)
                          # capture the armored CRC24 value
                          ^=(?P<crc>[A-Za-z0-9+/]{4})(?:\r?\n)
                          # finally, capture the armor tail line, which must match the armor header line
